app/components/autocomplete/at_mention/at_mention.js

```diff
@@ -0,0 +1,290 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React, {Component, PropTypes} from 'react';
+import {
+    ListView,
+    StyleSheet,
+    Text,
+    TouchableOpacity,
+    View
+} from 'react-native';
+import Icon from 'react-native-vector-icons/FontAwesome';
+
+import FormattedText from 'app/components/formatted_text';
+import ProfilePicture from 'app/components/profile_picture';
+
+import {makeStyleSheetFromTheme, changeOpacity} from 'app/utils/theme';
+
+const AT_MENTION_REGEX = /\B\@\w+$|\B\@$/i; // eslint-disable-line
```

Comment:
Shortened version worked great, thanks for that. Was getting some unnecessary escape characters thrown by eslint for the regex I was using. All good now that we're using the shortened version.